### DISCUSSION TITLE
Dynamic route deployment problems

### DISCUSSION THREAD

**Discussion by zjcwill** [ElMassimo/iles#17] [Q&A] [answered]
I used dynamic routing,When I build, the following files are generated.

![image](https://user-images.githubusercontent.com/22348984/138419038-4495becd-d60b-4b3c-b972-f4bb19322b4a.png)

How do I correctly point to `:id.html` when I visit the `/article/1` link

Do we have anything like this？https://nextjs.org/docs/api-reference/next.config.js/exportPathMap#adding-a-trailing-slash

**Comment by ElMassimo** [comment, marked as answer]
__Edit:__ @zjcwill `iles@0.4.2` added support for dynamic parameters + [`getStaticPaths`](https://iles-docs.netlify.app/guide/routing#getstaticpaths).

---

__Old Answer:__

Hi! At the moment îles does not support dynamic paths.

Planning to add a similar API soon, which might become available next week.

In the meantime, I would [recommend using Astro](https://iles-docs.netlify.app/faqs#astro) instead.

**Reply by zjcwill** [reply]
Hope to support dynamic routing soon

**Reply by ElMassimo** [reply]
Have a [working prototype](https://github.com/ElMassimo/iles/blob/dynamic/playground/the-vue-point/src/pages/posts/%5Bslug%5D.vue#L9), will announce it once it's released.
